src/Commands/BannedKeywordsBag.ts

```diff
@@ -0,0 +1,203 @@
+            "interface": false,
+            "internal": false,
+            "is": false,
+            "lambda": false,
+            "let": false,
+            "lock": false,
+            "log": false,
+            "log10": false,
+            "long": false,
+            "module": false,
+            "namespace": false,
+            "native": false,
+            "new": false,
+            "next": false,
+            "nil": false,
+            "not": false,
+            "null": false,
+            "number": false,
+            "object": false,
+            "of": false,
+            "open": false,
+            "operator": false,
+            "or": false,
+            "out": false,
+            "override": false,
+            "package": false,
+            "params": false,
+            "partial": false,
+            "pass": false,
+            "pi": false,
+            "print": false,
+            "private": false,
+            "protected": false,
+            "public": false,
+            "raise": false,
+            "range": false,
+            "readonly": false,
+            "redo": false,
+            "ref": false,
+            "remove": false,
+            "require": false,
+            "rescue": false,
+            "retry": false,
+            "return": false,
+            "sbyte": false,
+            "sealed": false,
+            "self": false,
+            "set": false,
+            "short": false,
+            "sin": false,
+            "sizeof": false,
+            "sqrt": false,
+            "stackalloc": false,
+            "static": false,
+            "strictfp": false,
+            "string": false,
+            "struct": false,
+            "super": false,
+            "switch": false,
+            "symbol": false,
+            "synchronized": false,
+            "tan": false,
+            "then": false,
+            "this": false,
+            "throw": false,
+            "throwas": false,
+            "throws": false,
+            "transient": false,
+            "true": false,
+            "try": false,
+            "type": false,
+            "typeof": false,
+            "uint": false,
+            "ulong": false,
+            "unchecked": false,
+            "undef": false,
+            "unless": false,
+            "unsafe": false,
+            "until": false,
+            "ushort": false,
+            "using": false,
+            "value": false,
+            "var": false,
+            "virtual": false,
+            "void": false,
+            "volatile": false,
+            "when": false,
+            "where": false,
+            "while": false,
+            "with": false,
+            "write": false,
+            "yield": false,
+            "zeros": false,
+        };
+    }
+
+    /**
+     * Returns whether the variable name is allowed.
+     * 
+     * @param name  The variable name
+     * @returns A boolean determining the validity of the variable name.
+     */
+    public validBoolean(alias: string): boolean {
```

Comment:
`validBoolean` is a weird name. Were you thinking of something like `validateName`?